ResearchKit.podspec

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |s|
+  s.name         = 'ResearchKit'
+  s.version      = '1.0.0'
```

Comment:
There's currently (no `1.0.0` tag)[https://github.com/ResearchKit/ResearchKit/tags], that would need to happen, too.